.github/workflows/deploy-hubs.yaml

```diff
@@ -81,6 +83,12 @@ jobs:
           (steps.config_files.outputs.hub_config == 'true')
         uses: mdgreenwald/mozilla-sops-action@v1
 
+      - name: Setup kops
+        if: |
+          (steps.base_files.outputs.files == 'true') ||
+          (steps.config_files.outputs.hub_config == 'true')
+        uses: hiberbee/github-action-kops@1.0.0
```

Comment:
This doesn't look like it's maintained anymore and its CI seems to be failing :(
But I agree that it's worth trying it and see if it works. In case it's working, maybe also leave a comment about the possibly of it failing in the future?

Reply:
The master version seems to be failing whereas 1.0.0 seems to be green... still, as you said, this is just a try before diving into our own kops install (if we need to do it).